docs/guide/images.md

```diff
@@ -356,4 +356,28 @@ ezpublish:
             provider: remote
             options:
                 url_pattern: 'http://example.com/var/site/storage/%%id%%'
-```
\ No newline at end of file
+```
+
+## Resizing images
+
+You can resize all images of a chosen Content Type using the `ezplatform:images:resize-original` command.
+You need to provide the command with:
+
+- identifier of the image Content Type
+- identifier of the Field you want to affect
+- name of the image variation to apply to the images
+
+`ezplatform:images:resize-original <Content Type identifier> <Field identifier> -f <variation name>`
+
+For example:
+
+`ezplatform:images:resize-original photo image -f small_image`
+
+Additionally you can provide two parameters:
+
+- `iteration-count` is the number of images to be recreated in a single iteration, to reduce memory use. Default is `25`.
+- `user` is the identifier of a user with proper permission who will perform the operation. Default is `admin`.
```

Comment:
TBH I don't know whether it's common in our documentation, but could you please list these permissions (`read`, `versionread`, `edit`, `publish`)?